pig/src/main/java/com/netflix/iceberg/pig/SchemaUtil.java

```diff
@@ -109,15 +118,21 @@ private static ResourceSchema convertComplex(Types.NestedField field) throws IOE
       case LIST:
         ListType listType = type.asListType();
 
-        ResourceSchema innerSchema = new ResourceSchema();
-        innerSchema.setFields(new ResourceFieldSchema[]{convert(listType.elementType())});
+        ResourceFieldSchema [] elementFieldSchemas = new ResourceFieldSchema[]{convert(listType.elementType())};
 
-        //Bag elements are tuples
-        ResourceFieldSchema tupleSchema = new ResourceFieldSchema();
-        tupleSchema.setType(DataType.TUPLE);
-        tupleSchema.setSchema(innerSchema);
+        //Wrap non-struct types in tuples
```

Comment:
minor: the comment seems to be for the else block

Reply:
Moved it.